lib/fizzy/parser_expr.cpp

```diff
@@ -95,7 +95,7 @@ inline bool type_matches(OperandStackType actual_type, OperandStackType expected
 /// @return optional type of the block result.
 parser_result<std::optional<ValType>> parse_blocktype(const uint8_t* pos, const uint8_t* end)
 {
-    // The byte meaning an empty wasm result type.
+    /// The byte meaning an empty wasm result type.
```

Comment:
Not sure if it makes sense to use doxygen comments inside functions? (it's an implementation detail after all)

But if it does, then next line should be doxygen, too.

Reply:
Yes, it shouldn't be in a function, for a second thought it was a struct. Will remove.

Reply:
Fixed.